backend/src/main/java/gov/cdc/usds/simplereport/service/ApiUserService.java

```diff
@@ -194,15 +194,16 @@ public UserInfo updateUserPrivileges(
 
   @AuthorizationConfiguration.RequirePermissionManageTargetUserNotSelf
   public UserInfo setIsDeleted(UUID userId, boolean deleted) {
-    ApiUser apiUser = getApiUser(userId);
+    ApiUser apiUser = getApiUser(userId, !deleted);
     apiUser.setIsDeleted(deleted);
     apiUser = _apiUserRepo.save(apiUser);
     _oktaRepo.setUserIsActive(apiUser.getLoginEmail(), !deleted);
     return new UserInfo(apiUser, Optional.empty(), isAdmin(apiUser));
   }
 
-  private ApiUser getApiUser(UUID id) {
-    Optional<ApiUser> found = _apiUserRepo.findById(id);
+  private ApiUser getApiUser(UUID id, Boolean includeArchived) {
```

Comment:
What do you think about leaving the `getApiUser(UUID id)` signature present, but just having it call `getApiUser(id, false)` to reduce the footprint of this overall change?